src/site/content/en/blog/samesite-cookie-recipes/index.md

```diff
@@ -0,0 +1,324 @@
+  explicitly mark their cookies for cross-site usage. Browsers are moving to
+  make cookies without a SameSite attribute act as first-party by default, a
+  safer and more privacy preserving option than the current open behavior.
+  Learn how to mark up your cookies to ensure your first-party and third-party
+  cookies continue to work once this change comes into effect.
+tags:
+  - post
+  - security
+  - cookies
+---
+
+{% Aside %}
+
+If you need a refresher on cookies and the `SameSite` attribute, head over to
+the earlier companion article,
+["SameSite cookies explained"](/samesite-cookies-explained).
+
+{% endAside %}
+
+To recap from before: [Chrome](https://www.chromium.org/updates/same-site),
+[Firefox](https://groups.google.com/d/msg/mozilla.dev.platform/nx2uP0CzA9k/BNVPWDHsAQAJ),
+[Edge](https://textslashplain.com/2019/09/30/same-site-cookies-by-default/), and
+others will be changing their default behavior so that:
+
+1. cookies without a `SameSite` attribute will be treated as `SameSite=Lax`, or
+   restricted to first-party contexts
+2. cookies for cross-site or third-party usage must specify
+   `SameSite=None; Secure`
+
+## Use cases for cross-site or third-party cookies
+
+There are a number of common use cases and patterns where cookies need to be
+sent in a third-party context. If you provide or depend on one of these use
+cases then you will need to ensure that either you or the provider are updating
+their cookies to ensure the service continues to function correctly.
+
+### Content within an `<iframe>`
+
+When content is displayed within an `<iframe>` this qualifies as a third-party
+context. Standard use cases here are:
+
+- embedded content shared from other sites, such as videos, maps, code samples,
+  social posts
+- widgets from external services such as payments, calendars, booking /
+  reservation services
+- widgets such as social buttons or anti-fraud services that create less obvious
+  `<iframes>`
+
+Cookies may be used here to personalize content if the current user has an
+existing account on the remote site, maintain a session state, store general
+preferences, enable usage statistics, and more.
+
+<figure class="w-figure  w-figure--center">
+  <img src="iframe.png"
+      alt="Diagram of a browser window where the URL of embedded content does
+        not match the URL of the page."
+      style="max-width: 35vw;">
+  <figcaption class="w-figcaption">
+    If the embedded content doesn't come from the same site as the top-level
+    browsing context, it's third-party content.
+  </figcaption>
+</figure>
+
+Additionally, as the web is inherently composable, `<iframes>` are used to embed
+content that is also viewed in a top-level or first-party context. Any cookies
+used by that site will be considered as third-party cookies when the site is
+displayed within the frame. If you're creating sites that you intend to be
+easily composed that also rely on cookies to function, you will also need to
```

Comment:
What is an easily composed site?

Reply:
Switched to "embedded" to hopefully make this more obvious.